### PR TITLE
projects page

### PR DESCRIPTION
**Pull request by laurelfan**
- projects page visible to all users, not just admins
- update styles to match @sfilman's mockup
  
- TODO make the links work
- TODO make project type work

### CODE REVIEW

**Comment by breville** [comment]
minor styling: maybe new paragraph for second sentence inside grey box?

File: dashboard/app/views/projects/projects.html.haml [hunk 1]
```diff
@@ -1,11 +1,45 @@
-- @page_title = 'Projects'
+- @page_title = t('project.projects')
 
 :scss
```
Comment (joshlory) [inline dashboard/app/views/projects/projects.html.haml]: Merge with application.css.scss?

File: dashboard/config/routes.rb [hunk 2]
```diff
@@ -72,12 +72,13 @@ def redirect_to_teacher_dashboard
...
-  get '/projects', to: 'projects#index'
-  get '/projects/:template', to: 'projects#template'
-
-  get '/p/artist', to: 'levels#show', key: 'New Artist Project', as: 'standalone_artist'
-  get '/p/playlab', to: 'levels#show', key: 'New Play Lab Project', as: 'standalone_playlab'
+  resources :projects, path: '/p/', only: [:index] do
+    collection do
+      get '/artist', to: 'levels#show', key: 'New Artist Project', as: 'artist'
+      get '/playlab', to: 'levels#show', key: 'New Play Lab Project', as: 'playlab'
+      get '/:template', to: 'projects#template'
```
Comment (joshlory) [inline dashboard/config/routes.rb]: Is this route still needed?

File: shared/js/angularProjects.js [hunk 3]
```diff
@@ -8,7 +8,7 @@
     'projectsApp.services'
   ]).config(['$routeProvider', function($routeProvider) {
     $routeProvider.when('/',
-                        {templateUrl: '/projects/projects',
+                        {templateUrl: '/p/projects',
```
Comment (joshlory) [inline shared/js/angularProjects.js]: Nit: line continuations are 4 spaces

File: shared/js/angularProjects.js [hunk 4]
```diff
@@ -51,6 +51,10 @@
 
     $scope.projects = projectsService.query();
 
+    //default sort order
```
Comment (joshlory) [inline shared/js/angularProjects.js]: Nit: `// Default ...`

File: dashboard/config/routes.rb (hunk 2 above)
Comment (laurelfan) [inline dashboard/config/routes.rb]: Yep

**Pull request merged**